tools/docker/ci/Dockerfile.openvino2021

```diff
@@ -0,0 +1,16 @@
+ARG OCCLUM_VERSION
+FROM occlum/occlum:$OCCLUM_VERSION-ubuntu18.04 as base
+LABEL maintainer="Sheng Gui <sheng.gui@intel.com>"
+
+WORKDIR /root
+RUN rm -rf /root/demos && \
+    git clone https://github.com/occlum/occlum.git && \
+    cp -r occlum/demos /root/demos && \
+    rm -rf /root/occlum
+
+WORKDIR /root/demos/openvino
+RUN bash -x install_cmake.sh && \
+    bash -x download_and_build_openvino_2021.sh && \
```

Comment:
download_and_build_openvino_2021.sh and download_openvino_model.sh should be used in demo_test.yml. It is the only way to test them in each commit.